gr-analog/grc/analog_probe_avg_mag_sqrd_x.xml

```diff
@@ -43,6 +57,12 @@
 		<name>in</name>
 		<type>$type.input</type>
 	</sink>
+	<source>
+		<name>out</name>
+		<type>$type.output</type>
+		<optional>$type.optional</optional>
+		<hide>$type.hide</hide>
```

Comment:
This did not work for me; GRC ignores this statement when the `type` enum changes. It's as if the cheetah templates are not being evaluated when the enum changes. I also tried the following to no avail:

`#if $type == 'cf' then 'none' else 'all' #`
`#if $type == 'cf' then False else True#`
`#if $type == 'cf' then False else True #`
`#if $type == 'cf' then none else all #`
`#if $type == cf then none else all #`
`#if $type == 'cf' then none else all #`
`{#if $type == 'cf' then 'none' else 'all'}`
`
#if $type == 'cf'
'none'
#else
'all'
#end if
`